integrations/pom.xml

```diff
@@ -221,7 +221,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>job-dsl</artifactId>
-      <version>1.66</version>
+      <version>1.72</version>
```

Comment:
Looks like this has a side affect of requiring jenkins version of `2.121` when the job-dsl plugin is installed.
Surprised your build passed =/

That update is fine since we were planning on updating it anyway, but a little confused

Reply:
because the integration module is already using 2.121 😄 
See https://github.com/jenkinsci/configuration-as-code-plugin/commit/4773f5e646c536f80973bc04ee7b4e46eeca25f3#diff-ad20f8808d77ba4b076cfd8eba6d97a9R15

Reply:
bahaha, well confusion solved, thanks